PWGCF/TwoParticleCorrelations/Tasks/lambdaR2Correlation.cxx

```diff
@@ -1029,6 +1061,14 @@ struct lambdaCorrelationAnalysis {
 
       histos.fill(HIST(sub_dir_recgen[rec_gen]) + HIST("h2d_n2_") + HIST(sub_dir_hist[part_pair]), rapphix + 0.5, rapphiy + 0.5, eff_1 * eff_2);
     }
+
+    // qinv histos
+    TLorentzVector q1, q2, q;
+    q1.SetXYZM(p1.px(), p1.py(), p1.pz(), MassLambda0);
+    q2.SetXYZM(p2.px(), p2.py(), p2.pz(), MassLambda0);
+    q = q1 - q2;
+    double qinv = TMath::Sqrt(q.M() * q.M());
```

Comment:
I would suggest start using the the methods in
https://github.com/AliceO2Group/O2Physics/blob/master/Common/Core/RecoDecay.h
instead of using `TLorentzVector`. It will be much more efficient